### DISCUSSION TITLE
关于多集视频的建议

### DISCUSSION THREAD

**Discussion by AstbReal** [Richasy/Bili.Uwp#578] [展示]
1.
每次在选集的时候都需要用滚轮选集滚很久，选集的旁边能否提供一个用鼠标拖动的进度条，这样可以更快捷方便的达到想要的位置？

2.能否提供一个选项：设置打开一个历史视频自动跳转到上次播放的位置。

谢谢博主大佬啦。

**Comment by Richasy** [comment]
新版本会提供索引视图

**Reply by AstbReal** [reply]
非常感谢
